ndk.rosinstall

```diff
@@ -682,15 +682,15 @@
     local-name: vision_opencv/cv_bridge
     uri: https://github.com/ros-gbp/vision_opencv-release/archive/release/kinetic/cv_bridge/1.12.8-0.tar.gz
     version: vision_opencv-release-release-kinetic-cv_bridge-1.12.8-0
-# - tar:
-#     local-name: vision_opencv/image_geometry
-#     uri: https://github.com/ros-gbp/vision_opencv-release/archive/release/kinetic/image_geometry/1.12.8-0.tar.gz
-#     version: vision_opencv-release-release-kinetic-image_geometry-1.12.8-0
-# - tar:
-#     local-name: vision_opencv/vision_opencv
-#     uri: https://github.com/ros-gbp/vision_opencv-release/archive/release/kinetic/vision_opencv/1.12.8-0.tar.gz
-#     version: vision_opencv-release-release-kinetic-vision_opencv-1.12.8-0
-# - tar:
-#     local-name: xacro
-#     uri: https://github.com/ros-gbp/xacro-release/archive/release/kinetic/xacro/1.11.3-0.tar.gz
-#     version: xacro-release-release-kinetic-xacro-1.11.3-0
+- tar:
+    local-name: vision_opencv/image_geometry
+    uri: https://github.com/ros-gbp/vision_opencv-release/archive/release/kinetic/image_geometry/1.12.8-0.tar.gz
+    version: vision_opencv-release-release-kinetic-image_geometry-1.12.8-0
+- tar:
+    local-name: vision_opencv/vision_opencv
+    uri: https://github.com/ros-gbp/vision_opencv-release/archive/release/kinetic/vision_opencv/1.12.8-0.tar.gz
+    version: vision_opencv-release-release-kinetic-vision_opencv-1.12.8-0
+- tar:
+    local-name: xacro
```

Comment:
xacro fails saying it doesn't find `roslint`. Please uncomment `roslint` as well from this list.

Reply:
I will comment again Xacro here, because I uncommented it by error.
Also, xacro works well if you uncomment the other dependencies needed.

Reply:
`xacro` is Python-based. It might work because all Python packages use the Python interpreter installed on the host. On Android it is useless.

Reply:
If there's no package depending on `xacro` (no package should depend on it at build time), please comment it out and add a comment saying it's python based.